scripts/ios/azure_ios_build_psv.sh

```diff
@@ -18,8 +18,9 @@
 # License-Filename: LICENSE
 
 # Getting rid of boost installed to the CI image (not clear when it appeared)
-# asciidoc and source-highlight are using it so dependencies are ignored
-brew uninstall --ignore-dependencies boost
+# asciidoc and source-highlight are using it so dependencies are ignored.
+# Sometimes there is no boost and there is no need to fail in this case.
+brew uninstall --ignore-dependencies boost || true
```

Comment:
Is this one related to the change?

Reply:
Partially. Recently there was an issue because of boost installed using brew in the image so it was uninstalled #1467. It appeared that not all containers has that boost so our builds started failing from time to time because of that. It was a separate commit in this PR, extracted to another PR #1471